index.html

```diff
@@ -171,69 +171,118 @@
       discovery mechanisms, link relations, errors and default language.
     </p>
     <p>
-      Future versions of this specification, or extension specifications, may 
+      Future versions of this specification, or extension specifications, may
       define additional Profiles.
     </p>
   </section>
 
   <section id='sotd'></section>
 
+  <section id="terminology">
```

Comment:
We should probably leave this where it is for consistency with the other WoT specifications, but could link the word "profile" in the introduction to its definition in the terminology section for clarity.

Reply:
I opened the discussion about it during today's main call. Cross linking is indeed a better solution.